### PR TITLE
Adds support for Qasa rental contracts

### PR DESCRIPTION
**Pull request by SatishBV**
# Jira Issue: [https://hedvig.atlassian.net/browse/APP-1591]

## What?
- Adds support for Qasa rental type contracts

## Optional checklist
- [ ] Unit tests written
- [ ] Tested locally
- [ ] Codescouted

### CODE REVIEW

File: src/resolvers/contracts/contracts.graphqls [hunk 1]
```diff
@@ -317,6 +318,27 @@ enum SwedishCarLineOfBusiness {
...
+  premium: MonetaryAmountV2!
+  certificateUrl: String
+  status: AgreementStatus!
+  address: Address!
+  type: SwedishQasaRentalType!
+  partner: String
+  carrier: String
+  monthlyRentalAmount: Int!
+  totalRentalAmount: Int!
+  refernceNo: String
```
Comment (AlexSJ1990) [inline src/resolvers/contracts/contracts.graphqls:334]: typo here in "reference"

**Review by cpiehl1 — COMMENTED** [review]
Looks good! I think some changes have to be made to `contractPills.ts` as well though!

File: src/cross-schema/detailstable/contractDetailsFragments.ts [hunk 2]
```diff
@@ -41,6 +41,18 @@ const agreementFragment = `
...
+    address {
+      street
+      zipCode: postalCode
+      city
+    }
+    squareMeters
+    activeFrom
+    coInsured: numberCoInsured
+    ancillarySpace: ancillaryArea
+    tenant
```
Comment (cpiehl1) [inline src/cross-schema/detailstable/contractDetailsFragments.ts:54]: This field doesn't seem to exist in the schema

File: src/test-utils/calls.ts [hunk 3]
```diff
@@ -75,6 +75,14 @@ export const contractFields = gql`
         registrationNumber
         mileage
       }
+      ... on SwedishQasaRentalAgreement {
+        numberCoInsured
+        squareMeters
```
Comment (cpiehl1) [inline src/test-utils/calls.ts:80]: Both of these ones are also missing

**Review by cpiehl1 — APPROVED** [review]
Nice!

**Pull request merged**